packages/extension-chakra-storefront/src/components/with-layout/with-layout.tsx

```diff
@@ -302,28 +308,40 @@ const withLayout = <P extends object>(WrappedComponent: React.ComponentType<P>)
                     </Box>
                     {!isOnline && <OfflineBanner />}
                     <AddToCartModalProvider>
-                        <SkipNavContent
-                            style={{
-                                display: 'flex',
-                                flexDirection: 'column',
-                                flex: 1,
-                                outline: 0
-                            }}
-                        >
+                        {!isBlocked ? (
```

Comment:
How about we be a bit more verbose? So it's clearer at first glance what is being blocked.

```suggestion
                        {!isNavigationBlocked ? (
```